contents/javascript/Javascript-framework-Redux.md

```diff
@@ -0,0 +1,226 @@
+<frontmatter>
+  title: Redux
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+  
+{{ booktitle | safe }}
+
+# Introduction to Redux
+
+**Author: [Labayna Neil Brian Narido](https://github.com/nbriannl)** <br>
+Reviewers: 
+
+## What is Redux?
+
+Simply put, Redux is library that manages application state for Javascript applications. You can use it together with view libraries such as React and Vue.
+
+More formally, the [official website](https://redux.js.org/introduction/getting-started) describes Redux as follows:
+
+> *Redux* is a predictable state container for JavaScript apps.
+>
+> It helps you write applications that behave consistently, run in different environments (client, server, and native), and are easy to test. On top of that, it provides a great developer experience, such as live code editing combined with a time traveling debugger.
+>
+> You can use Redux together with React, or with any other view library. It is tiny (2kB, including dependencies), but has a large ecosystem of addons available.
+
+### What is application state?
```

Comment:
Probably no need for this heading. The explanation flows better without the heading.